realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsManagedTests.java

```diff
@@ -530,7 +532,11 @@ public void migration_backlinkedFieldInUse() {
         } catch (IOException e) {
             fail("Failed copying realm");
         } catch (RealmMigrationNeededException expected) {
-            assertTrue(expected.getMessage().contains("Field count is"));
+            assertThat(expected.getMessage(),
+                    CoreMatchers.allOf(
+                            CoreMatchers.containsString("Property 'BacklinksSource.name' has been added"),
+                            CoreMatchers.containsString("Property 'BacklinksTarget.parents' has been removed")));
+            //assertTrue(expected.getMessage().contains("Field count is"));
```

Comment:
I don't think this has anything to do with ObjectStore and migration.  Please remove it.